nep-dapi.mediawiki

```diff
@@ -797,6 +797,90 @@ Example
 </pre>
 
 
+====verifyMessage====
+Returns whether the provided signature data matches the provided message and was signed by the account of the provided public key.
+
+
+'''Method Interface'''
+
+<pre>
+function verifyMessage({
+  message: string,
+  data: string,
+  publicKey: string,
+}): Promise<{result: boolean}>
+</pre>
+
+
+'''Input arguments'''
```

Comment:
for an Input arguments,

can there be another argument that take a hash of a message instead of passing the whole message in? 

something like
```js
interface VerifyMessageArgs {
  messageHash: string,
  signature: string
}
```